scripts/build.ps1

```diff
@@ -479,6 +449,71 @@ function PrintAndExit-OnError([System.String] $output){
     }
 }
 
+function Locate-MSBuildPath() {
```

Comment:
Please follow the coding convention. `{` on a new line. No `()` after a function.